tests/helpers/environment.js

```diff
@@ -33,6 +35,13 @@ class UnlockEnvironment extends PuppeteerEnvironment {
       1000 /* every s */,
       120 /* up to 2m */
     )
+    console.log(`Waiting for Subgraph at ${theGraphHost}:${theGraphPort}`)
```

Comment:
Now at least integration tests will not pass if the graph node is not running/accessible!